include/meta/meta_fwd.hpp

```diff
@@ -248,24 +258,29 @@ namespace meta
         typename T::type::value_type;
     }
     && Same<typename T::value_type, typename T::type::value_type>
+#if META_CXX_TRAIT_VARIABLE_TEMPLATES
```

Comment:
To me, this conditional is worth the trouble given that it's cheaper to instantiate a variable template than a class template. YMMV.

Reply:
But `is_integral_v<T>` is required to be defined as `is_integral<T>::value`. What exactly do we gain?

Reply:
It's not observable whether `is_foo_v` instantiates `is_foo` since `is_foo` isn't user-specializable.

Reply:
Hrrm, except that it is, because objects have addresses.